eth/ethconfig/config.go

```diff
@@ -36,6 +36,14 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
+const (
```

Comment:
I’d prefer to place the constant database name and metric namespace in their respective locations. Defining them here feels a bit out of place.